### DISCUSSION TITLE
VNM on MacOS problem with MODULEPATH

### DISCUSSION THREAD

**Discussion by TomEmotion** [neurodesk/neurodesk.github.io#143] [Q&A] [answered]
Running VNM on a MacBook pro as per the instructions. All went well, but in an open terminal window, module avail returns an error:

Lmod has detected the following error:  avail is not possible,
MODULEPATH is not set or not set with valid paths.

root@36e04b583208:~# export | grep MODULE
declare -x LMOD_DEFAULT_MODULEPATH="/vnm/containers/modules"
declare -x MODULEPATH="/vnm/containers/modules"
declare -x MODULESHOME="/usr/share/lmod/lmod"
root@36e04b583208:~# ls -l /vnm
total 0

**Comment by TomEmotion** [comment]
Not sure why all that text I added when I edited my comment is crossed out. Annoying...

**Comment by TomEmotion** [comment]
Started adding afni, and now the module avail command does not return an error. So perhaps caused by the module directory being completely empty when VNM first installed, in which case an easy fix to make sure something is in there when first installed?

**Comment by stebo85** [comment]
Dear @TomEmotion, yes - the error only comes if no packages is installed. I will add a check to suppress this error.

**Comment by stebo85** [comment, marked as answer]
Dear @TomEmotion, I added a check that now suppresses the error in case no package is available yet: 
![image](https://user-images.githubusercontent.com/4021595/103589882-32901b80-4f38-11eb-92a3-d11215b622a5.png)

can you test again in the latest version?:
docker run --privileged --name vnm -v ~/vnm:/vnm -e USER=neuro -p 6080:80 -p 5900:5900 vnmd/vnm:20210104
